src/lovok.cpp

```diff
@@ -1,16 +1,18 @@
 #include <cstdlib>
 #include "../include/lovok.h"
+#include "../libs/io/file_io.h"
 #include "lovok_handle_internal.h"
 
 
 LOVOK_HANDLE Lovok_create(const char *name) {
     LOVOK_HANDLE handle = NULL;
     handle = (LOVOK_HANDLE) malloc(sizeof(LOVOK_HANDLE_INTERNAL));
-    handle->name = name;
+    handle->wrapper = FileWrapper_Open(name);
     return handle;
 }
 
 void Lovok_destroy(LOVOK_HANDLE h) {
+    FileWrapper_Close(h->wrapper);
```

Comment:
Generally a good practice: Should check `h != nullptr` and make destroy functions a no-op for when that happens.